modules/osdk-ansible-watches-file.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
```

Comment:
We need to add a section on finalizers here @fabianvf is currently writing the docs for the repo right now.